<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCup</class>
 <widget class="QWidget" name="FCup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Исходящие формы в ЦУП</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/email_forward.png</normaloff>:/res/1/email_forward.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Даты</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Начало отбора</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de1">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Конец отбора</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de2">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbDat">
            <property name="text">
             <string>Включить отбор по датам</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Состояние формы:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="cb_notpr">
            <property name="text">
             <string>Не отправлена</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_otpr">
            <property name="text">
             <string>Отправлена</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_Refresh">
       <property name="text">
        <string>Обновить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbView">
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/text_align_justity.png</normaloff>:/res/1/text_align_justity.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDel">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/outlook_clean_up.png</normaloff>:/res/1/outlook_clean_up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/email_add.png</normaloff>:/res/1/email_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
